hopsworks-common/src/main/java/io/hops/hopsworks/common/featurestore/featuregroup/FeaturegroupController.java

```diff
@@ -619,8 +639,12 @@ private void verifyFeaturegroupUserInput(FeaturegroupDTO featuregroupDTO, Featur
   @TransactionAttribute(TransactionAttributeType.NEVER)
   public FeaturegroupDTO syncHiveTableWithFeaturestore(Featurestore featurestore, FeaturegroupDTO featuregroupDTO,
     Users user) throws FeaturestoreException {
-    //Verify basic feature group input information
-    verifyFeaturegroupUserInput(featuregroupDTO, featurestore);
+  
+    // Verify general entity related information
+    featurestoreInputValidation.verifyUserInput(featuregroupDTO, false);
```

Comment:
I assume here you wanted to set it to `true`. I'd still check the name of the feature, even for the "Hudi" tables. 
Since we want to have Hudi tables as default, we should be able to make them available online, hence we will have the same limitations in therms of column names.

Reply:
yes, right, here it should've been `true`. if we still want to check, I can remove the `sync` parameter all together

Reply:
removed it as discussed